pages/ksphere/kommander/1.1.0-beta/release-notes/index.md

```diff
@@ -50,6 +50,49 @@ Template:
 
 -->
 
+### Kommander 1.1.0 RC 4 - June 26th 2020
+
+| Kubernetes Support | Version |
+| ------------------ | ------- |
+| **Minimum**        | 1.15.0  |
+| **Maximum**        | 1.17.x  |
+| **Default**        | 1.17.3  |
+
+#### Features/Improvements
+* Add guideance what to do when cluster deletion failed
+* Allow default federated addon values to be overriden
+* Attach clusters to Kommander is now considered GA
+* Update kubecost web urls
+* Add flag (`--skip-credentials-display`) to Kommander to Konvoy it displaying login information in Logs
+* Add ability to disable federation of monitoring stack
+* Added autogenerated labels to advanced cluster create form
+* Improved context handling for multiple contexts when attaching cluster
+
+#### Bug Fixes
+* Kommander Grafana was unavailable after self-attaching host cluster as managed cluster
```

Comment:
Is this the detach Kommander bug? Was it only applicable to grafana? I thought all the ingresses stopped working?

Reply:
not 100% sure  tbh, thats what the JIRA was talking about. I know we fixed UI a while ago (probbaly before 1.0.0), not sure if grafana was missed or UI was the only one we fixed tho